bionic/tokenization.py

```diff
@@ -62,10 +65,17 @@ def tokenize(value, serialize_func=None):
     be appended.
     '''
 
+    # to-do: clean up temp_dir after use
+    # to-do: update docs
```

Comment:
We should probably do this one in this PR, to keep the docs in a consistent state.

In particular, I think the file you want to change is `docs/api/protocols.rst`, which has an example protocol with `read` and `write` methods.

Reply:
Sounds good, updated!

Reply:
Excellent, thank you!